packages/treat/src/webpack-plugin/treatCompiler.js

```diff
@@ -106,48 +99,86 @@ function getRootCompilation(loader) {
   return compilation;
 }
 
-function compileTreatSource(loader) {
+function compileTreatSource(loader, compat) {
   return new Promise((resolve, reject) => {
     // Child compiler will compile treat files to be evaled during compilation
     const outputOptions = { filename: loader.resourcePath };
 
     const childCompiler = getRootCompilation(loader).createChildCompiler(
       TWL,
       outputOptions,
-      [
-        new NodeTemplatePlugin(outputOptions),
-        new LibraryTemplatePlugin(null, 'commonjs2'),
-        new NodeTargetPlugin(),
-        new SingleEntryPlugin(loader.context, loader.resourcePath),
-        new LimitChunkCountPlugin({ maxChunks: 1 }),
-        new ExternalsPlugin('commonjs', 'treat'),
-      ],
     );
 
-    const subCache = 'subcache ' + __dirname + ' ' + loader.resourcePath;
```

Comment:
This cache doesn't seem to have been ported across?

Reply:
This was intentional. Not needed anymore.